imap_processing/__init__.py

```diff
@@ -4,12 +4,19 @@
 # packet definitions directory path.
 #
 # This directory is used by the imap_processing package to find the packet definitions.
+import os
 from pathlib import Path
 
+# This is where the file data is stored and organized by instrument and level.
+# The default location is in the current working directory, but can be
+# set on the command line using the --data-dir option, or through
+# the environment variable IMAP_DATA_DIR.
+DATA_DIR = Path(os.getenv("IMAP_DATA_DIR") or Path.cwd() / "imap-data")
+
 # Eg. imap_module_directory = /usr/local/lib/python3.11/site-packages/imap_processing
 imap_module_directory = Path(__file__).parent
 
-instruments = [
+INSTRUMENTS = [
```

Comment:
I changed these to a typical GLOBAL caps pattern since they seemed like globals. I can change these back if this is just noise.